torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2061,6 +2061,26 @@ def sample_inputs_broadcast_to(op_info, device, dtype, requires_grad, **kwargs):
             make_tensor(size, device, dtype, low=None, high=None, requires_grad=requires_grad),
             args=(shape,)) for size, shape in test_cases)
 
+def sample_inputs_broadcast_tensors(op_info, device, dtype, requires_grad, **kwargs):
+    make_arg = partial(make_tensor, dtype=dtype, device=device, requires_grad=requires_grad)
+    test_cases: Tuple[tuple] = (((3,), (1, 2, 1), (1, 1), (5, 1, 1),),)
+
+    def generator():
+        for shape, *other_shapes in test_cases:
+            yield SampleInput(make_arg(shape), args=tuple(make_arg(s) for s in other_shapes))
+
+    return list(generator())
```

Comment:
that's weird :p 
I would say that appending to a list take exactly the same number of lines and reduces A LOT the cognitive load when reading the code.

Reply:
Lol this is how most of the OpInfos are written. But since there isn't a pytorch style guide we should just go with whatever works... (append to list, generator approach, or list comprehension)

Reply:
I can understand the generator if the function does more things or is called twice with different parameters.
But here...

Reply:
Lol fixed!